src/main/java/pb360/controller/GenerateRestApiController.java

```diff
@@ -0,0 +1,35 @@
+package pb360.controller;
+
+import pb360.model.MessageObject;
+import pb360.service.GenerateRestApiService;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("v1/controllers")
+public class GenerateRestApiController {
+
+	@Autowired
+	private GenerateRestApiService generateRestApiService;
+
+	@RequestMapping(value = "/{restId}", method = RequestMethod.GET)
+	public HttpEntity<MessageObject> getRestApiDetails(@PathVariable("restId") String restId) {
+
+		MessageObject messageObj = generateRestApiService.getRestApiData(restId);
+
+		return new HttpEntity<MessageObject>(messageObj);
```

Comment:
help me change this code to return new ResponseEntity<MessageObject>(messageObj, HttpStatus.OK);

Reply:
I changed it to: return new ResponseEntity<MessageObject>(messageObj, HttpStatus.OK);
Please correct me if i'm wrong